src/ts/ConvivaAnalytics.ts

```diff
@@ -229,6 +229,31 @@ export class ConvivaAnalytics {
     }
   }
 
+  /**
+   * Puts the session state in a notMonitored state.
+   */
+  public pauseTracking(): void {
+    // AdStart is the right way to pause monitoring according to conviva.
+    this.client.adStart(
+      this.sessionKey,
+      Conviva.Client.AdStream.SEPARATE,
+      Conviva.Client.AdPlayer.SEPARATE,
+      Conviva.Client.AdPosition.PREROLL, // Also stops tracking time for VST so PREROLL seems to be sufficient
+    );
+    this.client.detachPlayer(this.sessionKey);
```

Comment:
shouldn't you update the playerStateManager here?

Reply:
No because the internal tracking doesn't stop but just don't get propagated to conviva. After the tracking is resumed the same instance of `playerStateManager` will be and should be used.